src/main/kotlin/net/emdal/tank/Node.kt

```diff
@@ -0,0 +1,20 @@
+package net.emdal.tank
+
+open class Node(val label: String) : Entity()
+
+fun <T : Node> Graph.node(
+  alias: String = "",
+  node: T,
+  block: T.() -> String = { "" }
+): Graph = node(node, alias, block)
+
+fun <T : Node> Graph.node(
+  node: T,
+  alias: String = "",
+  block: T.() -> String = { "" }
+): Graph {
+  return this.copy(
+    query = query + "($alias:${node.label} {${node.block()}})",
+    aliases = aliases + (alias to node)
+  )
+}
```

Comment:
I think these could be member functions of `Graph` instead of being extension functions.

Reply:
Rewrote it for that. Should add some Kdoc to the class/methods to explain the rationale behind having both of the methods?